src/FakeStateMachine.ts

```diff
@@ -13,12 +14,47 @@ interface State {
   Result?: Object;
   ResultPath?: string | null;
   Resource?: string;
-  Choices?: any[];
+  Choices?: TopLevelChoiceRule[];
   Default?: string;
   Next?: string;
   End?: boolean;
 }
-type Resource = { [key: string]: (arg: any) => any };
+
+export interface TopLevelChoiceRule extends ChoiceRule {
+  Next: string;
+}
+
+interface ChoiceRule extends Partial<BasicChoiceRule> {
+  And?: ChoiceRule[];
+  Or?: ChoiceRule[];
+  Not?: ChoiceRule;
+}
+
+interface BasicChoiceRule {
```

Comment:
[ask] Are there missing rules (e.g. `StringLessThan`)? If so, could you add TODO comments about missing rules?

https://states-language.net/spec.html#choice-state

For example, `// TODO: StringLessThan, ....`

Reply:
Fixed